mlflow/gateway/schemas/completions.py

```diff
@@ -95,7 +95,7 @@ class StreamChoice(ResponseModel):
 
 
 class StreamResponsePayload(ResponseModel):
-    id: str
+    id: Optional[str]
```

Comment:
Does pydantic automatically set this to None when id is unspecified?

Reply:
```suggestion
    id: Optional[str] = None
```

Can we explicitly set the default?

Reply:
Sure